fas2ipa/cli.py

```diff
@@ -296,6 +296,18 @@ def migrate_user(config, person, ipa):
         )
         try:
             ipa.user_add(person["username"], **user_args)
+            for group in person["memberships"]:
+                for agreement in config.get("agreement"):
+                    if group["name"] == agreement["signed_group"]:
```

Comment:
You could avoid one level of loop nesting if you prepared the group names list as:
```python
group_names = [g["name"] for g in person["memberships"]]
```
And the you could check with:
```python
if agreement["signed_group"] in group_names:
```
Both versions would work, it's my personal preference to avoid nesting levels when possible because I find it more readable, but you may have a different point of view.